### ISSUE TITLE
'checkbox-toggle' missing 'aria-describedby' set to input element

### ISSUE THREAD

**Issue by thowoo** [ng-lightning/ng-lightning#465]
Checkbox-toggle on the LDS site has 'aria-describedby' set to input element which bind to the <span id="....">, as below:
<input name="checkbox-toggle-2" type="checkbox" **aria-describedby="checkbox-toggle-2"** value="checkbox-toggle-2" />
    <span **id="checkbox-toggle-2"** class="slds-checkbox_faux_container" aria-live="assertive">

But ngl-checkbox-toggle does not have similar aria-describedby function which leads to accessibility problem.

![image](https://user-images.githubusercontent.com/31666497/56988496-2e560d00-6b5e-11e9-8ae0-bec5a8a4f519.png)

**Comment by bekos** [comment]
Nice catch @thowoo :) Will fix it soon, thx.
